app/src/main/java/com/keylesspalace/tusky/util/LogReader.kt

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2017 Andrew Dawson
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.util
+
+import android.content.Context
+import android.net.Uri
+import java.io.File
+import java.io.IOException
+
+
+object LogReader {
```

Comment:
Why the object? Just `getLogFile` a top level function

Reply:
because I copies it from Java version 😅